### PR TITLE
Fonts

### PR DESCRIPTION
**Pull request by tenzgass**
Add workspace
Define fonts

### CODE REVIEW

**Review by Simon-Busch — APPROVED** [review]

File: app/views/components/_card-workspace.html.erb [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="card-grid">
+    <a class="card" href="#">
+      <div class="card_background" ></div>
+      <div class="card_content">
+        <p class="card_category"><%= workspace.address %></p>
+        <h3 class="card_heading"><%= workspace.name %></h3>
+      </div>
+    </a>
+  </div>
+</section>
```
Comment (Simon-Busch) [inline app/views/components/_card-workspace.html.erb]: delete

File: app/views/workspaces/_card-workspace.html.erb [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+  <div class="card-grid">
+    <a class="card" href="#">
+      <div class="card_background"></div>
+      <div class="card_content">
+        <p class="card_category"><%= workspace.address %></p>
+        <h3 class="card_heading"><%= workspace.name %></h3>
+      </div>
+    </a>
+  </div>
+</section>
```
Comment (Simon-Busch) [inline app/views/workspaces/_card-workspace.html.erb]: Delete

File: app/views/workspaces/index.html.erb [hunk 3]
```diff
@@ -1,11 +1,14 @@
...
+      <div class="card_content">
+        <p class="card_category"><%= workspace.address %></p>
+        <h3 class="card_heading"><%= workspace.name %></h3>
+      </div>
+    </a>
+  </div>
 <% end %>
-</ul>
\ No newline at end of file
+</section>
```
Comment (Simon-Busch) [inline app/views/workspaces/index.html.erb]: keep here

**Review by Simon-Busch — APPROVED** [review]
Check comments

**Review by ericblandet — APPROVED** [review]

**Pull request merged**